src/client/Table.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import data from './data.json';
+
+function TableItem(props){
+  const isEven = (props.seq % 2 === 0) ? true : false;
+  const className = isEven ? 'table__row table__row--even' : 'table__row table__row--odd';
+
+  return(
+    <tr className={className}>
+        <td>{props.seq}</td>
+        <td>{props.status}</td>
+        <td>{props.category}</td>
+        <td>{props.title}</td>
+        <td>{props.owner}</td>
+        <td>{props.priority}</td>
+      </tr>
+  )
+}
+
+function ShowTable(props){
+  const tableData = props.tableData;
+  const tableItems = tableData.map((item) => 
+    <TableItem key={item.seq}
+                seq={item.seq} 
+                status={item.status}
+                category={item.category}
+                title={item.title}
+                owner={item.owner}
+                priority={item.priority} />
+    );
+  return (
+    <div className="table">
+      <table align="center" rules="all" className="table__main">
+        <tbody className="table__body">
+          <tr className="table__header">
```

Comment:
For basic structure of `table`, it contains `tbody` and `thead` which defines a set of rows defining the head and the body of the table respectively.
Therefore, I suggest to reposition your header into `thead` instead of `tbody`.

```js
<table align="center" rules="all" className="table__main">
  <thead className="table__head">
    // place your head of table here
  </thead>
  <tbody className="table__body"> { ... }</tbody>
</table>
``` 

ref: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead